hello-world.js

```diff
@@ -1,19 +1,24 @@
-import Component from "can-component";
-import DefineMap from "can-define/map/map";
-import stache from "can-stache";
+import { Component, DefineMap } from "can";
 
-const HelloWorldVM = DefineMap.extend({
-  visible: { default: false },
-  message: { default: "Hello There!" }
+const ViewModel = DefineMap.extend({
+  visible: {
+    default: false
+  },
+  message: {
+    default: "Hello There!"
+  }
 });
 
 Component.extend({
   tag: "hello-world",
-  view: stache("{{#if visible}}{{message}}{{else}}Click me{{/if}}"),
-  ViewModel: HelloWorldVM,
-  events: {
-    click: function() {
-      this.viewModel.visible = !this.viewModel.visible;
-    }
-  }
+  view: `
+  <div on:click="this.visible = not(this.visible)">
```

Comment:
Might be worth giving this block an extra level of indentation so it is visually inside the view line.